mantidimaging/gui/windows/main/presenter.py

```diff
@@ -29,6 +32,8 @@
 
 RECON_TEXT = "Recon"
 
+settings = QSettings('mantidproject', 'Mantid Imaging')
+
```

Comment:
I don't think this is used

Reply:
`settings` is used in `do_update_UI()`

```
    def do_update_UI(self) -> None:
        if settings.value('use_os_defaults', defaultValue='True') == 'True':
            extra_style = settings.value('extra_style_default')
            theme = 'Fusion'
            override_os_theme = 'False'
        else:
            extra_style = settings.value('extra_style')
            use_dark_mode = settings.value('use_dark_mode')
            theme = settings.value('theme_selection')
            override_os_theme = settings.value('override_os_theme')
```

Reply:
Sorry, I put the comment in the wrong place, should have been in `view.py` line 58. I think that one is not used

Reply:
Yes you're right, thats a remnant of me refactoring some things from the view to the presenter